tests/restmodel/test_container.py

```diff
@@ -28,11 +29,10 @@ def get_app(self):
         app = web.Application(handlers=handlers)
         app.file_config = Mock()
         app.file_config.network_timeout = 5
+        app.urlpath_for_object = Mock(return_value="/urlpath_for_object/")
```

Comment:
Yes good point, we need to test it.
But this is mocked there because we are using this generic web.Application in this test.
The reason is that we can't instantiate our application. We would need to mock a lot of stuff to do so and we are not ready for that yet.